pennylane/transforms/specs.py

```diff
@@ -63,22 +63,23 @@ def circuit(x, add_ry=True):
             return qml.probs(wires=(0,1))
 
     >>> qml.specs(circuit)(x, add_ry=False)
-    {'gate_sizes': defaultdict(int, {1: 1, 2: 1}),
-     'gate_types': defaultdict(int, {'RX': 1, 'CNOT': 1}),
-     'num_operations': 2,
-     'num_observables': 1,
-     'num_diagonalizing_gates': 0,
-     'num_used_wires': 2,
-     'depth': 2,
-     'num_trainable_params': 1,
-     'num_device_wires': 2,
-     'device_name': 'default.qubit',
-     'diff_method': 'parameter-shift',
-     'expansion_strategy': 'gradient',
-     'gradient_options': {'shift': 0.7853981633974483},
-     'interface': 'autograd',
-     'gradient_fn': 'pennylane.gradients.parameter_shift.param_shift',
-     'num_gradient_executions': 2}
+    {'resources': Resources(num_wires=2, num_gates=2, gate_types=defaultdict(<class 'int'>, {'RX': 1, 'CNOT': 1}), depth=2, shots=0),
```

Comment:
@Jaybsoni @soranjh - in retrospect, we might have needed a ticket to update uses of `specs` in the docs.